src/app/downloads/downloads.component.css

```diff
@@ -1,27 +1,45 @@
 :host {
   position: relative;
   display: flex;
-  padding: 40px 20px 10px 20px;
+  padding: 40px 20px 20px 20px;
+  min-height: 500px;
 }
 .controls {
-  flex: 1 1 30%;
-  max-width: 30%;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 10px;
+  flex-grow: 4;
+  flex-shrink: 2;
+  flex-basis: 20%;
+  padding-right: 20px;
 }
 .content {
-  position: relative;
-  flex: 7 7 70%;
-  max-width: 70%;
+  flex-grow: 8;
+  flex-shrink: 6;
+  flex-basis: 80%;
+  max-width: 80%;
+}
+@media only screen
+and (max-device-width: 1024px) {
+  .content {
+    max-width: 70%
+  }
+}
+@media only screen
+and (max-device-width: 768px)
+and (min-device-width: 569px) {
+  .content {
+    max-width: 60%
+  }
 }
-.bar {
-  width: 75%;
-  padding: 0 40px;
-  margin: 20px 0px;
-  border: 1px solid #a3a3a3;
+@media only screen
+and (max-device-width: 568px) {
```

Comment:
responsive 🌮 ! 
left-align looks slightly odd with nothing to its right -- could center the controls in their container in this rule? (or can wait for user testing, i don't trust my own opinions) 

![image](https://user-images.githubusercontent.com/16007382/31904287-d2fd442e-b7f8-11e7-89c2-32fd21bd136c.png)

Reply:
I have some different ideas about the mobile layout too like hiding controls in a menu and when open putting date controls to the left and interval and episodes on the right (if that would fit.) I'm imagining that we'll actually get some mobile specific tickets out of this meeting when we have it https://github.com/PRX/metrics.prx.org/issues/28. I just added this bit of media queries because Peter was looking at the app on smaller screen sizes already and commented about it, so this should just make it less bad.

Reply:
👍